src/drivers/CMakeLists.txt

```diff
@@ -12,4 +12,6 @@ if(CONFIG_DW)
 	add_subdirectory(dw)
 endif()
 
+add_subdirectory(generic)
+
```

Comment:
'generic' is a pretty generic term. We should come up with a better name for this.

Reply:
I moved it to `drivers/generic` due to Liam's review.